myuw/dao/student_profile.py

```diff
@@ -5,6 +5,15 @@
 
 from uw_sws.person import get_person_by_regid
 from myuw.dao.pws import get_regid_of_current_user
+from myuw.dao.enrollment import (get_current_quarter_enrollment,
+                                 get_main_campus,
+                                 get_all_enrollments,
+                                 get_majors_for_terms,
+                                 get_minors_for_terms)
+from myuw.dao.term import (get_current_quarter,
+                           get_next_quarter,
+                           get_current_and_next_quarters)
+from myuw.dao.gws import is_grad_student
 
 
 def get_profile_of_current_user():
```

Comment:
I believe this is also utilized by the  IASystem DAO https://github.com/uw-it-aca/myuw/blob/develop/myuw/dao/iasystem.py#L22-L24